sdk/react/src/useDVCInitialized.ts

```diff
@@ -0,0 +1,24 @@
+import { useContext, useState } from 'react'
+import context from './context'
+
+export const useDVCInitialized = (): boolean => {
+    const [isDVCReady, setIsDVCReady] = useState(false)
+    if (isDVCReady) return isDVCReady
+
+    const dvcContext = useContext(context)
```

Comment:
Should this line and the undefined check be moved to the top of the function so React will render the same way on every call?

Reply:
Yup, I'll update :) the linter also caught this!